Mailer/SymfonyAuthCodeMailer.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Scheb\TwoFactorBundle\Mailer;
+
+use Scheb\TwoFactorBundle\Model\Email\TwoFactorInterface;
+use Symfony\Component\Mailer\Mailer;
+use Symfony\Component\Mime\Email;
+
+class SymfonyAuthCodeMailer implements AuthCodeMailerInterface
+{
+    /**
+     * @var Mailer
+     */
+    private $mailer;
+
+    /**
+     * @var string
+     */
+    private $senderEmail;
+
+    /**
+     * @var string|null
+     */
+    private $senderName;
+
+    public function __construct(Mailer $mailer, string $senderEmail, ?string $senderName)
+    {
+        $this->mailer = $mailer;
+        $this->senderEmail = $senderEmail;
+        $this->senderName = $senderName;
```

Comment:
This is not used anywhere

Reply:
A nice approach is to directly create a named address.
No need for 2 properties when only one is needed.

```php
use Symfony\Component\Mime\Address;

$this->senderAddress = new Address($senderEmail,  $senderName);
```